todo_app/app.py

```diff
@@ -1,11 +1,31 @@
-from flask import Flask
+from flask import Flask, render_template, request
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
-@app.route('/')
+@app.route('/',methods = ['POST', 'GET'])
 def index():
-    return 'Hello World!'
+    if request.method == 'POST':
+        title = request.form.get('title')
+        if title:
+            new_item = add_item(title)
+        else:
+            pass
```

Comment:
There's no need to put an else statement if you're just going to skip that case, you're fine to just have:
```suggestion
        if title:
            new_item = add_item(title)
```